react/features/base/config/functions.js

```diff
@@ -58,6 +58,21 @@ const WHITELISTED_KEYS = [
      */
     'callUUID',
 
+    /**
+     * The UUID of the CallKit call representing the conference/meeting
+     * associated with this config.js. The property is meant for use cases in
+     * which Jitsi Meet is to work with a CallKit call created outside of Jitsi
+     * Meet and to be adopted by Jitsi Meet such as, for example, an incoming
+     * and/or outgoing CallKit call created by Jitsi Meet SDK for iOS
+     * clients/consumers prior to giving control to Jitsi Meet. As the value is
+     * associated with a conference/meeting, the value makes sense not as a
+     * deployment-wide configuration, only as a runtime configuration
+     * override/overwrite provided by, for example, Jitsi Meet SDK for iOS.
+     *
+     * @type string
+     */
+    'callHandle',
```

Comment:
Our coding style calls for alphabetical sorting for easy dictionary-like scanning by the reader. The purpose of the sorting is exactly with such big lists in mind. Please make sure `callHandle` is sorted accordingly.